.github/workflows/buildAndTestDatabaseIteratorsStandalone.yml

```diff
@@ -22,12 +22,12 @@ jobs:
         python-version: 3.9
 
     - name: Checkout project
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         path: sandbox
 
     - name: Install Ninja
-      uses: llvm/actions/install-ninja@55d844821959226fab4911f96f37071c1d4c3268
+      uses: lukka/get-cmake@latest
```

Comment:
I am not sure if we'll want to change to some third-party action. I looked at the action upstream and it could be a single-character change. I'll look into that...